ansible-ipi-install/roles/node-prep/templates/install-config.j2

```diff
@@ -23,6 +23,9 @@ platform:
 {% endif %}
 {% if bootstraposimage is defined and bootstraposimage|length %}
     bootstrapOSImage: {{ bootstraposimage }}
+{% endif %}
+{% if (release_version[0]|int > 4) or ((release_version[0]|int == 4) and (release_version[2]|int > 3)) and (clusterosimage is defined and clusterosimage|length) %}
```

Comment:
We can simplify this and leave off the version check because it turns out that 4.3 does support `clusterOSImage`.  Just flagging it here -- I know we already discussed this and I don't intend to nag!

Reply:
will fix.